CHANGELOG_UNRELEASED.md

```diff
@@ -0,0 +1,2 @@
+### Fixed
+- Fixed bug that shows duplicated icon with user position when positioning is active
```

Comment:
- Fixed bug that doesnt clean the previous user position in some cases after calling presentInNewNiew